lib/logstasher/log_subscriber.rb

```diff
@@ -62,6 +61,11 @@ def extract_request(payload)
         :path       => extract_path(payload),
         :route      => "#{payload[:controller]}##{payload[:action]}"
       }
+      metadata = ::LogStasher.metadata
+      if !!metadata
```

Comment:
This will always be true because of `config.logstasher.metadata  = {}` in `railtie.rb`.

`!!{} #=> true`

Reply:
You can also say:
```
result.merge!(:metadata => metadata) unless metadata&.empty?
```